src/main/java/frontend/AdvancedTab.java

```diff
@@ -0,0 +1,35 @@
+package frontend;
+
+import javax.swing.*;
+import java.awt.*;
+
+class AdvancedTab extends JComponent {
```

Comment:
I see now that we're planning on accessing these through keys. Ignore this thread unless there is some edge case here that demands attention.